scapy/layers/dhcp.py

```diff
@@ -111,12 +119,28 @@ def answers(self, other):
 
 
 class _DHCPParamReqFieldListField(FieldListField):
-    def getfield(self, pkt, s):
-        ret = []
-        while s:
-            s, val = FieldListField.getfield(self, pkt, s)
-            ret.append(val)
-        return b"", [x[0] for x in ret]
+    def randval(self):
+        class _RandReqFieldList(RandField):
+            def _fix(self):
+                return [RandByte()] * int(RandByte())
+        return _RandReqFieldList()
+
+
+class RandClasslessStaticRoutesField(RandField):
+    """
+    A RandValue for classless static routes
+    """
+
+    def _fix(self):
+        return "%s/%d:%s" % (RandIP(), RandByte(), RandIP())
```

Comment:
```suggestion
        return "%s/%d:%s" % (RandIP(), RandNum(0, 32), RandIP())
```

Reply:
When I wrote this I was wondering what would happen on common implementations if you passed a number > 32, but you're right we aren't fuzzing here.